src/js/views/filters/FilterGroupView.js

```diff
@@ -72,6 +75,47 @@ define(['jquery', 'underscore', 'backbone',
           //Create a ToggleView
           var filterView = new ToggleFilterView({ model: filter });
         }
+        else if (filter.get("fields").includes("sem_annotation") && MetacatUI.appModel.get("bioportalAPIKey")) {
+          
+          var view = this;
+
+          var filterView = new AnnotationFilterView({
+            selected: filter.get("values"),
+            inputLabel: filter.get("label"),
+            separatorText: view.model.get("operator"),
+            placeholderText: filter.get("placeholder"),
+            icon: filter.get("icon"),
+            subType: "AnnotationFilterView",
+            useSearchableSelect: true
+          });
+
+          filterView.off("annotationSelected");
+          filterView.on("annotationSelected", function(event, item){
+            // Get the value of the associated input
+            var term = (!item || !item.value) ? input.val() : item.value;
+            var label = (!item || !item.filterLabel) ? null : item.filterLabel;
+
+            //set up annotation label mappings
+            var annotationLabelsMappings;
+            if(filter.get("annotationLabelsMappings")) {
+              annotationLabelsMappings = _.clone(filter.get("annotationLabelsMappings"));
+            }
+            else {
+              annotationLabelsMappings  = new Object();
+            }
+            annotationLabelsMappings[term] = label;
+            filter.set("annotationLabelsMappings", annotationLabelsMappings);
+
+            var newValues = _.clone(filter.get("values"));
+            // append to values array if not exist
+            if (newValues.indexOf(term) == -1) {
+              newValues.push(term);
+              filter.set({"values": newValues});
+            }
+
+            view.trigger("addNewAnnotationSearch", event, item, filter);
+          });
+        }
```

Comment:
This works fine but I thought I'd point out that this logic could be encapsulated within a AnnotationFilter model to make it line up better with the rest of the logic so then it could just be `var filterView = new AnnotationFilterView({ model: filter });`. Part of why this is the way it is is because the AnnotationFilterView lives in pre-Portal-era times so it's serving multiple roles.

@laurenwalker let me know if you feel strongly about this